hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/assumed_roles.md

```diff
@@ -153,6 +153,14 @@ Here are the full set of configuration options.
   </description>
 </property>
 
+<property>
+  <name>fs.s3a.assumed.role.external.id</name>
+  <value />
```

Comment:
can you an a valid example

Reply:
yes.

Reply:
I think putting an example here is meaningless, because external id is just a string set by the user, it can by any value. No unified format.